cnnlabs/cnn-hapi
# Feature/service start stop

- Addresses issue #16 
- Allows users to start / stop services such as a cache store

To test:

- spin up a standalone Redis process
- spin up the example app
- you should see the redis connection take place `onPreStart` (called before the connection listeners are started)

- send the `SIGINT` event
- the redis connection should quit `onPostStop` (called after the connection listeners are stopped)
